### PR TITLE
Create interface xml

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -1513,6 +1513,35 @@ def create_controller_xml(contr_dict):
     return contr
 
 
+def create_iface_xml(iface_dict):
+    """
+    Create interface xml file
+
+    :param iface_dict: The dict params includes interfaces' configurations
+
+    :return: the interface object
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py:1522]: change to the interface object representing iface xml ?

**Comment by chunfuwen** [comment]
Any result available?

**Comment by chunfuwen** [comment]
./libvirt/tests/src/virtual_network/iface_options.py:52:    def create_iface_xml(iface_mac):

Previously there exists one method in tp-libvirt, is it possible to make them compatible ? Maybe in the future we can remove the one defined in tp-libvirt.

**Comment by dzhengfy** [comment]
As there is no big requirement to add this, I decide to close it.

**Pull request closed without merging**